scripts/bootstrap.sh.in

```diff
@@ -163,16 +165,12 @@ sync_salt() {
         salt-run saltutil.sync_roster saltenv=metalk8s-@@VERSION
 }
 
-sync_local() {
```

Comment:
Was not used?

Reply:
Was used BUT it's already done in `configure_salt_minion_local_mode` and it's the only time where we want to sync locally